themes/powerline-naked/powerline-naked.theme.bash

```diff
@@ -17,6 +17,9 @@ USER_INFO_SSH_CHAR=${POWERLINE_USER_INFO_SSH_CHAR:=" "}
 USER_INFO_THEME_PROMPT_COLOR=${POWERLINE_USER_INFO_COLOR:=240}
 USER_INFO_THEME_PROMPT_COLOR_SUDO=${POWERLINE_USER_INFO_COLOR_SUDO:=202}
 
+TERRAFORM_THEME_PROMPT_COLOR=${POWERLINE_TERRAFORM_COLOR:=161}
+TERRAFORM_CHAR=${POWERLINE_TERRAFORM_CHAR:="❲t❳ "}
+
```

Comment:
Looks like this somehow got duplicated in this file?

Reply:
removed duplicated code